runtime-manager/modules/ROOT/pages/rtm-traffic-inspection.adoc

```diff
@@ -0,0 +1,111 @@
+= Traffic Inspection for Standalone Mules
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords: agent, runtime manager, traffic inspection, standalone
+:page-deployment-options: hybrid
+
+The Traffic Inspection feature for Mules adds support in the agent for a forward proxy that is deployed in customers' premises and acts as a man in the middle between the Mule and the control plane by intercepting and inspecting all HTTPS traffic. 
+
+== Prerequisites 
+
+=== Traffic Inspection Proxy
+
+* An HTTP Inspection Proxy is required, with support for TLS connections to the runtime client, and mTLS connections to the control plane server.
+* The Inspection Proxy server does not require client authentication from the agents. The proxy does not require the Mule Agent to present a client certificate. Communication between the Mule Agent and the inspection proxy is TLS, not mTLS.
+* The inspection proxy must be provisioned to send a BofA private certificate to the Mule Agent. The Mule Agent uses a Certificate Authority from the JVM's keystore to validate the public certificate presented by the inspection proxy.
+* The inspection proxy and the MuleSoft control plane communicate via mTLS. There are two certificates involved:
+** The control plane presents a MuleSoft public server certificate to the inspection proxy. The proxy must be provisioned with the correct Certificate Authority to validate the server certificate presented by the MuleSoft control plane
+** The control plane requires a client certificate from the inspection proxy. This certificate must be provided by the customer to MuleSoft. 
++
+[NOTE]
+Communication with the control plane fails if the certificate does not match the specified serial number and common name.
+
+== Mule Installation and Registration Steps
+
+=== Provision the Truststore of the JVM with the Proxy Root CA
+
+. Identify the folder location of the Java Virtual Machine.
+. Insert the Root CA of the proxy in the truststore of the mule's JVM: in your terminal window, run the following command, replacing `$JAVA_HOME` with the actual path:
++
+[source,console,linenums]
+----
+sudo keytool -import -alias testCert -keystore $JAVA_HOME/jre/lib/security/cacerts -file proxy_cacert.pem
+----
++
+. Enter the provided password.
+. If you have multiple versions of Java, insert the certificate in the version that is being used by the Mule runtime.
+
+[[install-mule-runtime]]
+=== Install the Mule Runtime
+
+. Install the latest available Mule runtime version. You can skip this step if already installed.
++
+To check the latest Mule Runtime version, see xref:release-notes::mule-runtime/mule-esb.adoc[].
```

Comment:
```suggestion
To check the latest Mule version, see xref:release-notes::mule-runtime/mule-esb.adoc[].
```

Reply:
Check https://confluence.internal.salesforce.com/display/MTDT/Term+Presentation+Reference for how to refer to Mule.